.github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Preguntar sobre el encajamiento software para revisión
+about: ¿No estas seguro·a que tu paquete encaja para revisión?Usa esta plantilla para obtener repuesta de les editores en Español (Experimental)
+---
+
+
+Persona Encargada: nombre (@github_handle)  
+Repositorio:   
+
+---
+
+
+
+-   Pega el archivo DESCRIPTION completo dentro del siguiente bloque de código.
+
+```
+
+```
+
+
+## Alcance 
+
+- Por favor, indica qué categoría(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés)](https://ropensci.github.io/dev_guide/policies.html#package-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
+
+	- [ ] recuperación de datos (data retrieval)
+	- [ ] extracción de datos (data extraction)
+	- [ ] munging de datos (data munging)
+	- [ ] disposición o declaración de datos (data deposition)
+	- [ ] automatización de flujos de trabajo (workflow automation)
+	- [ ] control de versiones (version control)
+	- [ ] manejo de citas y bibliometría (citation management and bibliometrics)
+	- [ ] envoltorios de software científico (scientific software wrappers)
+	- [ ] herramientas para trabajo de campo y reproducibilidad (field and lab reproducibility tools)
+	- [ ] ligamientos con software de base de datos (database software bindings)
+	- [ ] datos geoespaciales (geospatial data)
+	- [ ] análisis de texto (text analysis)
+
+- Explica cómo y por qué el paquete encaja dentro de estas categorías (1 a 3 oraciones):
```

Comment:
Should "éstas categorías" have a tilde in "estas"? I never learned this rule :(

Reply:
Well, neither did I :cry: In any case if there's a mistake here we need to correct the other Spanish template as well.